src/extensions/epigenomics/Encode.ttl

```diff
@@ -44,206 +41,11 @@ DSPCore:hasEthnicity
-  <http://datamodel.terra.bio/DSPCore#hasDataModality> <http://datamodel.broadinstitute.org/DSPCoreValueSets#MicroRNACounts> ;
-  rdfs:label "ENCFF213BVA" ;
-  skos:prefLabel "ENCFF213BVA" ;
-.
-Encode:ENCFF424QYJ
-  a <http://datamodel.terra.bio/DSPCore#File> ;
-  DSPCore:hasCrossReference "https://www.encodeproject.org/files/ENCFF424QYJ/"^^dcterms:URI ;
-  DSPCore:hasFormat "bed" ;
-  <http://datamodel.terra.bio/DSPCore#hasDataModality> <http://datamodel.broadinstitute.org/DSPCoreValueSets#MicroRNACounts> ;
-  rdfs:label "ENCFF424QYJ" ;
-  skos:prefLabel "ENCFF424QYJ" ;
-.
-Encode:ENCFF488IYR
-  a <http://datamodel.terra.bio/DSPCore#File> ;
-  DSPCore:hasCrossReference "https://www.encodeproject.org/files/ENCFF488IYR/"^^dcterms:URI ;
-  DSPCore:hasFormat "rcc" ;
-  <http://datamodel.terra.bio/DSPCore#hasDataModality> <http://datamodel.broadinstitute.org/DSPCoreValueSets#MicroRNACounts> ;
-  rdfs:label "ENCFF488IYR" ;
-  skos:prefLabel "ENCFF488IYR" ;
-.
-Encode:ENCFF660GHH
-  a <http://datamodel.terra.bio/DSPCore#File> ;
-  DSPCore:hasCrossReference "https://www.encodeproject.org/files/ENCFF660GHH/"^^dcterms:URI ;
-  DSPCore:hasFormat "rcc" ;
-  <http://datamodel.terra.bio/DSPCore#hasDataModality> <http://datamodel.broadinstitute.org/DSPCoreValueSets#MicroRNACounts> ;
-  rdfs:label "ENCFF660GHH" ;
-  skos:prefLabel "ENCFF660GHH" ;
-.
-Encode:ENCLB011GAL
-  a DSPCore:Library ;
-  DSPCore:hasCrossReference "http://www.encodeproject.org/libraries/ENCLB011GAL/"^^dcterms:URI ;
-  Encode:hasAlias "ali-mortazavi: human-adrenal-gland-ENTEX52-Lib2" ;
-  Encode:hasLab "http://www.encodeproject.org/labs/ali-mortazavi/" ;
-  Encode:strandSpecificity false ;
-  Encode:submittedBy "http://www.encodeproject.org/users/bc5b62f7-ce28-4a1e-b6b3-81c9c5a86d7a/" ;
-  dcterms:created "2016-07-25T22:17:41.258151+00:00" ;
-  rdfs:label "ENCLB011GAL" ;
-  skos:prefLabel "ENCLB011GAL" ;
-.
-Encode:ENCSR895SFC
-  a Encode:Experiment ;
-  DSPCore:dateReleased "2016-08-01T00:00:00"^^xsd:dateTime ;
-  DSPCore:hasCrossReference "http://www.encodeproject.org/experiments/ENCSR895SFC"^^dcterms:URI ;
-  DSPCore:usesSample Encode:ENCBS719AMH ;
-  <http://datamodel.terra.bio/DSPCore#hasAssay> Encode:Assay_for_ENCSR895SFC ;
-  Encode:hasLibrary Encode:ENCLB011GAL ;
-  Encode:hasStatus "released" ;
-  dcterms:created "2016-07-25T22:17:18.167352+00" ;
-  dcterms:dateSubmitted "2016-07-31" ;
-  rdfs:label "ENCSR895SFC" ;
-  skos:prefLabel "ENCSR895SFC" ;
-  prov:hadActivity Encode:MortazaviLab_NanoStringPipeline ;
-  prov:used Encode:ENCBS719AMH ;
-.
-Encode:EncodeDataset
-  a DSPdcat_ap:Dataset ;
-  DSPdcat_ap:hasPrimaryConsent DSPCore:NoRestriction ;
-  dcterms:hasPart Encode:ENCBS562VSE ;
-  dcterms:hasPart Encode:ENCBS719AMH ;
-  dcterms:hasPart Encode:ENCD0451RUA ;
-  rdfs:label "EncodeDataset" ;
-  skos:prefLabel "EncodeDataset" ;
-.
 Encode:Experiment
   a owl:Class ;
   rdfs:label "Experiment" ;
   rdfs:subClassOf <http://datamodel.terra.bio/DSPCore#Activity> ;
```

Comment:
```suggestion
  rdfs:subClassOf DSPCore:Activity ;
```
I saw that you added a a prefix for this address at the beginning of the file - it can be used here, right?
Same thing for EncodeExamples.ttl at lines 24, 269

Reply:
Yes, good catch.  Fixing in next version.